src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_LENGT = 20;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        switch (FigureTipe.values()[random.nextInt(FigureTipe.values().length)]) {
```

Comment:
Introduce `random.nextInt(FigureTipe.values().length)` as local variable `index`
and `FigureTipe.values()[...]` as `figure`